x-pack/plugins/ml/server/models/data_frame_analytics/analytics_manager.ts

```diff
@@ -35,11 +36,21 @@ export class AnalyticsManager {
   private _client: IScopedClusterClient['asInternalUser'];
   private _mlClient: MlClient;
   public _inferenceModels: TrainedModelConfigResponse[];
+  public _jobStats: DataFrameAnalyticsStats[];
```

Comment:
there is a convention that only `private` elements are prefixed with `_`. You have a setter and getter, hence should be `private`.
```suggestion
  private _jobStats: DataFrameAnalyticsStats[];
```

Reply:
Good catch - updated in 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9